CHANGELOG.md

```diff
@@ -3,6 +3,46 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.6.0
+
+_Feb 10, 2022_
+
+A big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎛 [`FilterPanel` content customization](https://mui.com/components/data-grid/filtering/#customize-the-filter-panel-content) via props (#3497) @alexfauquette
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.6.0` / `@mui/x-data-grid-pro@v5.6.0`
+
+#### Changes
+
+- [DataGrid] Add `debounceMs` option to `setEditCellValue` method (#3825) @m4theushw
+- [DataGrid] Allow to translate `checkboxSelection` labels (#3846) @m4theushw
+- [DataGrid] Customize `FilterPanel` with props (#3497) @alexfauquette
+- [DataGrid] Fix filtering of string columns for value=0 (#3843) @flaviendelangle
+- [DataGrid] Fix focus when blur event rerenders the grid (#3718) @alexfauquette
+- [DataGridPro] Add clear error when the tree data has duplicated paths (#3840) @flaviendelangle
+- [DataGridPro] Avoid imports from `@mui/base` (#3903) @cherniavskii
+- [DataGridPro] Register column pinning after selection (#3887) @m4theushw
+- [l10n] Improve Turkish (trTR) locale (#3842) @flaviendelangle
+
+### Docs
+
+- [docs] Clarify a bit the migration to v5 (#3847) @oliviertassinari
+- [docs] Fix sorting feature link (#3877) @alexfauquette
+- [docs] Migrate content to the new location (#3730) @siriwatknp
```

Comment:
AFAIK it's not rolled out to production yet

Reply:
Ok, I thought it would be in production as soon as you will deploy the doc at the end of the release 

@flaviendelangle do you know when it will be live?